cmd/validator/slashing-protection/export.go

```diff
@@ -67,6 +67,22 @@ func exportSlashingProtectionJSON(cliCtx *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "could not export slashing protection history")
 	}
+
+	// Check if JSON data is empty and issue a warning about common problems to the user.
+	if eipJSON == nil || len(eipJSON.Data) == 0 {
+		log.Warnf(
```

Comment:
Wouldn't fatal for (2) be more appropriate here? but i guess it's hard to separate them out